src/HashSetHashMap/HashSet/ContainsDuplicate/Solutions/Solution0001.java

```diff
@@ -0,0 +1,51 @@
+package HashSetHashMap.HashSet.ContainsDuplicate;
+
+import org.junit.*;
+import static org.junit.Assert.*;
+import java.util.*;

```

Comment:
I think it would be best to clean up the imports and just import everything like this.